tests/common.py

```diff
@@ -570,6 +570,13 @@ def load_fixture(filename: str, integration: str | None = None) -> str:
     return get_fixture_path(filename, integration).read_text(encoding="utf8")
 
 
+async def async_load_fixture(
+    hass: HomeAssistant, filename: str, integration: str | None = None
+) -> str:
```

Comment:
Note: I was unable to use this in evil_genius_labs, as it uses "package" scoped fixtures, which causes ScopeMismatch exceptions

https://github.com/home-assistant/core/blob/dddeafa28867e9e37a6eca3824167daa6f629305/tests/components/evil_genius_labs/conftest.py#L22-L26